python/hail/methods/statgen.py

```diff
@@ -2918,46 +2920,55 @@ def _local_ld_prune(mt, call_field, r2=0.2, window=1000000, memory_per_core=256)
+        require_biallelic(mt, 'ld_prune')._jvds, call_field, float(r2), bp_window_size, max_queue_size))
 
     return sites_only_table
 
 
 @typecheck(call_expr=expr_call,
            r2=numeric,
-           window=int,
+           bp_window_size=int,
            memory_per_core=int)
-def ld_prune(call_expr, r2=0.2, window=1000000, memory_per_core=256):
+def ld_prune(call_expr, r2=0.2, bp_window_size=1000000, memory_per_core=256):
     """Prune variants in linkage disequilibrium.
 
-    .. include:: ../_templates/req_unphased_diploid_gt.rst
+    .. include:: ../_templates/req_diploid_gt.rst
 
     .. include:: ../_templates/req_biallelic.rst
 
     .. include:: ../_templates/req_tvariant.rst
 
     Notes
     -----
-
-    This method prunes variants in linkage disequilibrium in two stages. The first stage
-    is a local pruning step, which prunes variants in the same partition. The parallelism
-    in this step will be affected by the number of partitions in the dataset, as well as
-    the memory per core, which is used to calculate a queue size for the local prune.
-
-    The second stage is a global pruning step which makes use of a correlation matrix
-    across all remaining variants. In the global pruning step, correlated variants are
-    passed to a method that computes a maximal independent set of those variants.
+    This method removes variants in linkage disequilibrium to ensure that
+    the squared Pearson correlation coefficient :math:`r^2` of any pair of
+    variants at most `window_size` base pairs apart is less than or
+    equal to `r2`. Each variant is represented as a vector over samples
+    with elements given by the (mean-imputed) number of alternate alleles.
+    In particular, even if present, **phase information is currently ignored**.
+
+    The method proceeds in two stages. The first stage is a local pruning step,
+    which prunes variants in the same partition. The parallelism in this step
+    will be affected by the number of partitions in the dataset, as well as
+    the memory per core, which is used to calculate a queue size for local
+    pruning.
+
+    The second stage is a global pruning step which computes the correlation
+    matrix across all remaining variants. In the global pruning step, correlated
+    variants are passed to a method that computes a maximal independent set of
+    those variants.
 
     Parameters
     ----------
     call_expr : :class:`.CallExpression`
-        Entry-indexed call expression on a matrix table with row-indexed variants and column-indexed samples.
+        Entry-indexed call expression on a matrix table with row-indexed
+        variants and column-indexed samples.
     r2 : :obj:`float`
-        correlation threshold (exclusive) above which variants are removed
-    window : :obj:`int`
-        distance in kilobases; correlation between variants further apart is not considered in pruning
+        Squared correlation threshold.
```

Comment:
Can you add back the part about it being `exclusive`? Also, this is inconsistent with what is in the docs above. Can you double check whether it's inclusive or exclusive?

Reply:
Good catch. It's wasn't clear to me whether exclusive meant with respect to pruning or keeping. I read it the wrong way, so I've fixed the docs to say "strictly less" and added `(exclusive upper bound)`.